x-pack/plugins/cases/common/types/api/case/v1.test.ts

```diff
@@ -461,6 +462,102 @@ describe('CasesFindRequestRt', () => {
   });
 });
 
+describe('CasesSearchRequestRt', () => {
+  const defaultRequest = {
+    tags: ['new', 'case'],
+    status: CaseStatuses.open,
+    severity: CaseSeverity.LOW,
+    assignees: ['damaged_racoon'],
+    reporters: ['damaged_racoon'],
+    defaultSearchOperator: 'AND',
+    from: 'now',
+    page: '1',
+    perPage: '10',
+    search: 'search text',
+    searchFields: ['title', 'description'],
+    to: '1w',
+    sortOrder: 'desc',
+    sortField: 'createdAt',
+    owner: 'cases',
+    customFields: {
+      toggle_custom_field_key: [true],
+      another_custom_field: [false],
+    },
+  };
+
+  it('has expected attributes in request', () => {
+    const query = CasesSearchRequestRt.decode(defaultRequest);
+
+    expect(query).toStrictEqual({
+      _tag: 'Right',
+      right: { ...defaultRequest, page: 1, perPage: 10 },
+    });
+  });
+
+  it('removes foo:bar attributes from request', () => {
+    const query = CasesSearchRequestRt.decode({ ...defaultRequest, foo: 'bar' });
+
+    expect(query).toStrictEqual({
+      _tag: 'Right',
+      right: { ...defaultRequest, page: 1, perPage: 10 },
+    });
+  });
+
+  describe('errors', () => {
+    it('throws error when invalid searchField passed', () => {
+      expect(
+        PathReporter.report(
+          CasesFindRequestRt.decode({ ...defaultRequest, searchFields: 'foobar' })
```

Comment:
This is inside `describe('CasesSearchRequestRt'`, should this test and the ones below be `CasesFindRequestRt`? 😅

Reply:
yeah updated